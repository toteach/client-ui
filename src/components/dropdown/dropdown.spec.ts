import { mount, Wrapper } from '@vue/test-utils';
import Vue from 'vue';

import TDropdown from './dropdown.vue';

describe('TDropdown', () => {
  let wrapper: Wrapper<Vue>;
  beforeEach(() => {
    wrapper = mount(TDropdown, {});
  });
  it('is initialized', () => {
    expect(wrapper.name()).toBe('TDropdown');
    expect(wrapper.isVueInstance()).toBeTruthy();
  });
  it('renders correctly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
