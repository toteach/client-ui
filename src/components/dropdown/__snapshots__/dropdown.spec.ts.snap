// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TDropdown renders correctly 1`] = `
"<div class=\\"t-dropdown\\">
  <div role=\\"button\\" class=\\"t-dropdown__trigger\\"></div>
  <!---->
</div>"
`;
